src/mame/efo/cedar_magnet.cpp

```diff
@@ -31,50 +31,49 @@
 
 */
 
-
 /*
 
- Magnet System by
-
- EFO SA (Electrónica Funcional Operativa SA).
- based on Cedar hardware
-
+ Magnet System by EFO SA (Electrónica Funcional Operativa SA).
+ Based on Cedar hardware
 
  http://retrolaser.es/cedar-computer-el-ordenador-profesional-de-efo-sa/
  http://www.recreativas.org/magnet-system-2427-efosa
 
  A number of original games as well as conversions were advertised for this system, it is however
- believed that EFO went bankrupt before anything hit the market.  The only 3 dumped games are
- conversions and appear to be in incomplete states (it is rather easy to break Time Scanner for
- example, the ball simply gets stuck in some places)  These are not simply bootlegs, they're
- completely original pieces of code more akin to home computer ports.
+ believed that EFO went bankrupt before anything hit the market. The conversions are not simply
+ bootlegs, they're completely original pieces of code more akin to home computer ports.
 
  The following were advertised
   Original Games
   - A Day in Space ** *
   - Crazy Driver
+  - Formula
   - Jungle Trophy
-  - Quadrum
-  - War Mission ** *
-  - The Burning Cave *
-  - Scorpio
   - Paris Dakar ** *
+  - Quadrum
   - Sailing Race
-  - Formula
+  - Scorpio
+  - The Burning Cavern *
+  - War Mission ** *
 
   Ports / Conversions
-  - Exzisus *
+  - Boody Kids *
```

Comment:
Didn't you add this as "Boo**b**y Kids"?

Reply:
Fixed, thanks!